devenv/docker/blocks/tempo/docker-compose.yaml

```diff
@@ -1,20 +1,3 @@
-x-logging: &default-logging
-  driver: loki
-  options:
-    loki-url: 'http://localhost:3100/api/prom/push'
-    labels: namespace
-    loki-relabel-config: |
```

Comment:
don't we need the relabel config anymore?

Reply:
I don't think so, this is just devenv. It created a job and instance label, which I don't see a particular reason we need to keep. It was basically for demo purposes.

Reply:
Don't we use these as defaults for trace to logs tags?

Reply:
I could add back in the `job` target_label. But can leave out `instance`, no? It's mentioned in the provisioning example (so is job) but don't see it in our spans resource attributes like job. Would make it easier if job was back in actually as then it would be a Loki label and a trace span attr.

Reply:
ok sorry I thought we use these labels here as defaults: https://github.com/grafana/grafana/blob/0db397b8be0a0b894fcc96d115ff8ffa172b6894/public/app/features/explore/TraceView/createSpanLink.tsx#L118 but we use different labels. So I assume these are not really needed (at least not for the trace to logs)